pywren/pywren_ibm_cloud/storage/storage_utils.py

```diff
@@ -68,6 +68,32 @@ def create_output_key(prefix, executor_id, callgroup_id, call_id):
     return os.path.join(prefix, executor_id, callgroup_id, call_id, output_key_suffix)
 
 
+def create_group_prefix(prefix, executor_id, callgroup_id):
```

Comment:
@omerb01 the logic of this method is only used in `get_calls_ids()` (storage.py). I think it would be better to move this logic there.

Reply:
thanks, will move it to  storage.py